zephyr/esp32/CMakeLists.txt

```diff
@@ -109,6 +109,8 @@ if(CONFIG_SOC_SERIES_ESP32)
     ../../components/wpa_supplicant/src/utils
     ../../components/esp_coex/include
     ../../components/mbedtls/port/include
+
+    ../port/include/boot
```

Comment:
Only ESP32-S3 have this `../port/include/boot` moved to this area due to AMP support. Anyway, should we perform the same change for all SoCs in order to keep it all a bit in sync?